### PR TITLE
Download file

### PR DESCRIPTION
**Pull request by nhannn-2212**

### CODE REVIEW

**Comment by nhannn-2212** [comment]
Description: Download file
Ticket: https://edu-redmine.sun-asterisk.vn/issues/24416
image

**Comment by nhannn-2212** [comment]
image

**Comment by nhannn-2212** [comment]
ready

File: app/views/documents/_document.html.erb [hunk 1]
```diff
@@ -1,8 +1,11 @@
 <li id="document-<%= document.id %>">
   <span class="name"><%= document.name %></span>
   <% if document.doc.content_type == Settings.pdf_type %>
-    <span class="view"><%= link_to t("view"), document %>
+    <span class="view"><%= link_to t("view"), document %></span>
   <% end %>
+  <span class="download">
+    <%= link_to t("download"), download_path(:id => document.id), target: "_blank" %>
```
Comment (vanvtt-0952) [inline app/views/documents/_document.html.erb:7]: `download_path(:id => document.id)` thành `download_path(id: document.id)` hoặc `download_path(document)`

**Comment by nhannn-2212** [comment]
ready

**Pull request merged**